addons/account/wizard/account_invoice_send.py

```diff
@@ -86,7 +86,9 @@ def _send_email(self):
         if self.is_email:
             self.composer_id.send_mail()
             if self.env.context.get('mark_invoice_as_sent'):
-                self.mapped('invoice_ids').write({'invoice_sent': True})
+                #Salesman send posted invoice, without the right to write
+                #but the should have the right to change this flag
```

Comment:
```suggestion
                #but they should have the right to change this flag
```

Reply:
Plus it would be cleaner to put a space after the hashtag too.